include/ttmlir/Dialect/TTKernel/IR/TTKernelOpsEnums.td

```diff
@@ -7,22 +7,60 @@
 
 include "mlir/IR/EnumAttr.td"
 
-def TTKernel_Noc0 : I32EnumAttrCase<"Noc0", 0, "noc0">;
-def TTKernel_Noc1 : I32EnumAttrCase<"Noc1", 1, "noc1">;
-def TTKernel_Tensix : I32EnumAttrCase<"Tensix", 2, "tensix">;
-def TTKernel_Ethernet : I32EnumAttrCase<"Ethernet", 3, "ethernet">;
+def TTKernel_Noc : I32EnumAttrCase<"Noc", 0, "noc">;
+def TTKernel_Tensix : I32EnumAttrCase<"Tensix", 1, "tensix">;
+def TTKernel_Ethernet : I32EnumAttrCase<"Ethernet", 2, "ethernet">;
```

Comment:
This was an error previously, but we should prefix these enum def's with the enum type they belong to just to follow tablegen standard guidelines and to avoid naming collisions.  So:

```
def TTKernel_ThreadTypeNoc : ...
def TTKernel_ThreadTypeTensix : ...
def TTKernel_ThreadTypeEthernet : ...
```